### ISSUE TITLE
TerraWDIOSpecReporter path in Docs is incorrect

### ISSUE THREAD

**Issue by TheSavageDev** [cerner/terra-toolkit-boneyard#47]
# Bug Report

## Description

In the [TerraWDIOSpecReporter](https://github.com/cerner/terra-toolkit-boneyard/blob/main/docs/TerraWDIOSpecReporter.md) docs the example has the incorrect path to the file.

Should be `terra-toolkit/reporters/wdio/TerraWDIOSpecReporter`

## Steps to Reproduce

1. Tried to add as per the example and received an error in eslint

## Additional Context / Screenshots

![Screen Shot 2020-07-30 at 13 21 09](https://user-images.githubusercontent.com/67115428/88959402-8ef35e80-d267-11ea-9ef5-3b9a756c3f2b.png)

## Expected Behavior

The example to be correct.

## Possible Solution

Fix the path to the file.

## Environment

* Component Name and Version: terra-toolkit v.6.6.0
* Browser Name and Version: N/A
* Node/npm Version: [e.g. Node 8/npm 5] Node lts/dubnium
* Webpack Version: N/A
* Operating System and version (desktop or mobile): macOS v. 10.15.5 Catalina
